dev-tools/action-install-minikube.sh

```diff
@@ -30,10 +30,26 @@ if [[ "$OSTYPE" == "linux-gnu"* ]]; then
 elif [[ "$OSTYPE" == "darwin"* ]]; then
         # Mac OSX
         PLATFORM=darwin-amd64
-        echo "unsupported operating system; ignoring fluvio install"
+        
+        curl -LO https://storage.googleapis.com/minikube/releases/${MINIKUBE_VERSION}/minikube-${PLATFORM}
+        sudo install minikube-${PLATFORM} /usr/local/bin/minikube
+
+        # Install hyperkit using brew for osx minikube driver
+        brew install hyperkit
+
+        # Start Minikube with `hyperkit` driver
+        # OSx does not have `none` bare metal option
+        sudo minikube start --driver=hyperkit -p minikube
+
+        # Download kubectl
+        sudo minikube kubectl -- get po -A
+
+        # Run Minikube Tunnel
+        sudo nohup minikube tunnel >/tmp/tunnel.out 2>/tmp/tunnel.out &
```

Comment:
minikube tunnel is not mandatory for local cluster.  so make this optional?

Reply:
Keeping this to satisfy load balancer error.